src/util/durations.ts

```diff
@@ -0,0 +1,41 @@
+import durationsConfig, { type DurationConfig } from '../durations.config'
+
+/** Initialize duration helper. */
+function init() {
+  let inTest: boolean = !!navigator.webdriver
+
+  /** Returns the duration for a particular key or undefined if it doesn't exist. */
+  const get = (key: string): number => {
```

Comment:
Since the duration keys can be determined at compile-time, we can use Typescript to restrict the type here instead of doing a runtime check. This will provide better DX through autosuggest as well.

Reply:
I think I got this right (I at least got the error to go away...)